widgetssdk/src/main/java/com/glia/widgets/view/unifiedui/exstensions/ViewExstensions.kt

```diff
@@ -130,6 +130,7 @@ internal fun View.applyLayerTheme(layer: LayerTheme?) {
 }
 
 internal fun MaterialCardView.applyCardLayerTheme(layer: LayerTheme?) {
+    layer?.fill?.primaryColor?.also { if (Color.alpha(it) > 0) cardElevation = 0f }
```

Comment:
Could you please explain the logic here? If the color has opacity then we remove card elevation? What's the purpose of this?